modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -56,163 +52,166 @@ For example:
+* Review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+* Provision additional worker nodes as needed to maintain availability of applications when applying appliance upgrades.
 +
 If the amount of available resources for application deployments is less than the resources of one worker node, add another worker node with the same amount of CPU, memory, and disk resources to Runtime Fabric.
 +
 See xref:manage-nodes.adoc[How to Add a Node to Runtime Fabric].
-. To upgrade Runtime Fabric components:
 
-.. Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.  
-.. Navigate to your Runtime Fabric by selecting it on the *Runtime Fabrics* tab in Runtime Manager.
-.. Select "Upgrade to vx.x". The status of the Runtime Fabric is updated to show that the upgrade procedure is in progress.
-.. Wait for the status to transition to *Active*. 
+=== Upgrade Runtime Fabric Components
+
+. Navigate to your Runtime Fabric by selecting it on the *Runtime Fabrics* tab in Runtime Manager.
+. Select *Upgrade to vx.x*. 
++
+The status of the Runtime Fabric is updated to show that the upgrade procedure is in progress.
+. Wait for the status to transition to *Active*. 
 +
 In most cases, the upgrade takes fewer than 10 minutes. When the status transitions to *Active* and the version of the Runtime Fabric reflects the new version, the component upgrade is complete.
 +
 If the Runtime Fabric and Runtime Manager connection disconnects during the upgrade, connectivity is restored after the upgrade finishes.
-+
-Keep your browser session open to use in later steps.
-+
-[IMPORTANT]
-To upgrade to Runtime Fabric version 1.5 or later, you must be running version 1.4.54. If you are not running version 1.4.54, a button to upgrade to 1.4.54 is displayed. Upgrade to this version in order to enable upgrades to 1.5.
-. If you are not using the latest version of the `rtfctl` command-line utility, download the latest version:
-.. Using your terminal on a controller node, run the following command: 
+. Keep your browser session open to use in later steps.
+
+=== Upgrade the `rtfctl` Command-line Utility
+
+. If you are not using the latest version of the `rtfctl` command-line utility, download the latest version with the following command:
 +
 ----
 cd /opt/anypoint/runtimefabric
 sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
 ----
-+
-.. Change file permissions for the `rtfctl` binary: 
+. Change file permissions for the `rtfctl` binary: 
 +
 ----
 sudo chmod +x rtfctl
 ----
-. If you are not using the latest version of the appliance, upgrade to the latest version:
+. Alternatively, to update `rtfctl` in place as a user who owns the `rtfctl` binary, run:
```

Comment:
I did not comment on this in DOCS-11300 add topic for upgrading RTF-SMK #385 and I probably should have: is this truly an alternative, or are there two different approaches for two different types of users?
It might be a bit difficult to translate "...update `rtfctl` in place as a user who owns the `rtfctl` binary."

Reply:
good call. Should just simplify this to: 

```suggestion
. If you are a user who owns the `rtfctl` binary, you can update `rtfctl` without needing to download a new version. To do so, run the following command:
```